content/en/docs/KRE/user/60_kais_runner_sdk/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "KAI Server Runner SDK"
```

Comment:
Overall, I think we need a section that provides a real example of what the handler looks like now that we have to listen to subscriptions. In particular, it would be interesting to get an example of a handler that listens to 2+ streams and how that's handled. Both in Go (I think we have an example below) and Python.

Reply:
That's something users can have a look at in our demo and training repos. However, I have added a more simplified version for python regarding previous suggestion. An example on how to load functions should illustrate how messages are filtered, but a real logic I think it is better to have it at demo and training repos.